assets/src/domain/eventEditor/ui/EventEditor.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { DatesList } from './datetimes/datesList';
+import { FormModal } from '../../shared/ui/formModal';
+import { TicketsList } from './tickets/ticketsList';
+import { toast } from 'react-toastify';
+
+import { useEditorInitialization } from '../hooks';
+import { useStatus, TypeName } from '@appServices/apollo/status';
+
+import 'react-toastify/dist/ReactToastify.css';
+import './styles.scss';
+
+toast.configure({
+	autoClose: 3000,
+	className: 'ee-toaster-notice__container',
+	hideProgressBar: true,
+});
```

Comment:
May be move the toaster init to somewhere in `/application` as `initToast.ts`, so that it can be reused in other domains if needed?

Reply:
it could just go into `/application/services/toaster` with the other toaster files

Reply:
done, thx